lapce-ui/src/plugin.rs

```diff
@@ -155,36 +177,77 @@ impl Plugin {
         let size = ctx.size();
         let padding = 10.0;
 
-        let text = if status == PluginStatus::Install {
-            status.to_string()
+        if status == PluginStatus::Install {
+            let text_layout = ctx
+                .text()
+                .new_text_layout(status.to_string())
+                .font(config.ui.font_family(), config.ui.font_size() as f64)
+                .text_color(
+                    config
+                        .get_color_unchecked(LapceTheme::EDITOR_BACKGROUND)
+                        .clone(),
+                )
+                .build()
+                .unwrap();
+            let text_size = text_layout.size();
+            let text_padding = 5.0;
+            let x = size.width - text_size.width - text_padding * 2.0 - padding;
+            let y = y + self.line_height * 2.0;
+            let color = Color::rgb8(80, 161, 79);
+            let rect =
+                Size::new(text_size.width + text_padding * 2.0, self.line_height)
+                    .to_rect()
+                    .with_origin(Point::new(x, y));
+            ctx.fill(rect, &color);
+            ctx.draw_text(
+                &text_layout,
+                Point::new(
+                    x + text_padding,
+                    y + text_layout.y_offset(self.line_height),
+                ),
+            );
+            rect
         } else {
-            format!("{} ▼", status)
-        };
-        let text_layout = ctx
-            .text()
-            .new_text_layout(text)
-            .font(config.ui.font_family(), config.ui.font_size() as f64)
-            .text_color(
-                config
-                    .get_color_unchecked(LapceTheme::EDITOR_BACKGROUND)
-                    .clone(),
-            )
-            .build()
-            .unwrap();
-        let text_size = text_layout.size();
-        let text_padding = 5.0;
-        let x = size.width - text_size.width - text_padding * 2.0 - padding;
-        let y = y + self.line_height * 2.0;
-        let color = Color::rgb8(80, 161, 79);
-        let rect = Size::new(text_size.width + text_padding * 2.0, self.line_height)
-            .to_rect()
-            .with_origin(Point::new(x, y));
-        ctx.fill(rect, &color);
-        ctx.draw_text(
-            &text_layout,
-            Point::new(x + text_padding, y + text_layout.y_offset(self.line_height)),
-        );
-        rect
+            let color = match status {
+                PluginStatus::Installed => LapceTheme::EDITOR_FOCUS,
+                PluginStatus::Install => LapceTheme::EDITOR_FOREGROUND,
```

Comment:
pointless checking for `PluginStatus::Install` since you check for it higher
```rs
if status == PluginStatus::Install {
```

remove it with e.g. `PluginStatus::Disabled` and use `_ => LapceTheme...`

Reply:
<img width="584" alt="image" src="https://user-images.githubusercontent.com/41265098/190868248-cf5bcc76-0a4e-4408-8e2c-a465f59f82e1.png">
Like this ?

Reply:
```rs
            let color = match status {
                PluginStatus::Installed => LapceTheme::EDITOR_FOCUS,
                PluginStatus::Upgrade(_) => LapceTheme::LAPCE_WARN,
                PluginStatus::Disabled | _ => LapceTheme::EDITOR_DIM,
            };
```
or
```rs
            let color = match status {
                PluginStatus::Installed => LapceTheme::EDITOR_FOCUS,
                PluginStatus::Upgrade(_) => LapceTheme::LAPCE_WARN,
                _ => LapceTheme::EDITOR_DIM,
            };
```

1st one would need clippy lint override

Reply:
I've use the second one.